src/main/java/no/nav/familie/integrasjoner/personopplysning/PersonopplysningerController.kt

```diff
@@ -35,31 +35,45 @@ class PersonopplysningerController(private val personopplysningerService: Person
                 .body(ikkeTilgang("Ikke tilgang mot personopplysning ${e.message}"))
     }
 
+    @GetMapping(produces = [MediaType.APPLICATION_JSON_VALUE], path = ["aktorId/{tema}"])
+    fun aktørId(@RequestHeader(name = "Nav-Personident") personIdent: String,
+                @PathVariable tema: Tema): ResponseEntity<Ressurs<List<String>>> {
+        return ResponseEntity.ok().body(success(data = personopplysningerService.hentAktørId(personIdent, tema.toString()),
+                                                melding = "Hent aktørId OK"))
+    }
+
+    @GetMapping(produces = [MediaType.APPLICATION_JSON_VALUE], path = ["identer/{tema}"])
+    fun identer(@RequestHeader(name = "Nav-Personident") personIdent: String,
+                @PathVariable tema: Tema): ResponseEntity<Ressurs<List<IdentInformasjon>>> {
+        return ResponseEntity.ok().body(success(data = personopplysningerService.hentIdenter(personIdent, tema.toString()),
+                                                melding = "Hent aktørId OK"))
+    }
+
     @GetMapping(produces = [MediaType.APPLICATION_JSON_VALUE], path = ["v1/historikk"])
-    fun historikk(@RequestHeader(name = "Nav-Personident") @NotNull personIdent: String?,
-                  @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @NotNull fomDato: LocalDate?,
-                  @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @NotNull tomDato: LocalDate?)
+    fun historikk(@RequestHeader(name = "Nav-Personident") personIdent: String,
+                  @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) fomDato: LocalDate,
+                  @RequestParam @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) tomDato: LocalDate)
```

Comment:
En liten ting; ble det postet noe på Slack med tanke på bruk av personIdent som header - at det er litt "fy-fy" og at de ønsket at vi bruker `POST` med personIdent i body for å unngå at den blir logget?

https://nav-it.slack.com/archives/CJN0STWB0/p1588154696123900

Reply:
Ja. Det var meningen å endre det, men så glemte jeg det. I'll fix